ElrondNetwork/api.elrond.com
# Account transfers endpoint

## Type

## Problem setting
- Currently there is no possibility to show smart contract results and transactions in same place
  
## Proposed Changes
- Unify transactions and smart contract results within transfers endpoint

## How to test
- Make request to /accounts/:address:/transfers add see if there are transactions and scrs (by type)
- Apply filters to transfers requests
